src/ezdxf/explode.py

```diff
@@ -140,11 +144,15 @@ def virtual_block_reference_entities(block_ref: 'Insert', uniform_scaling_factor
     """
     assert block_ref.dxftype() == 'INSERT'
     Ellipse = cast('Ellipse', factory.cls('ELLIPSE'))
+    if skipped_entity_callback is None:
+        def skipped_entity_callback(_entity):
+            pass
 
-    def disassemble(layout):
+    def disassemble(layout) -> Generator['DXFGraphic', None, None]:
         for entity in layout:
             dxftype = entity.dxftype()
             if dxftype == 'ATTDEF':  # do not explode ATTDEF entities
+                skipped_entity_callback(entity)
```

Comment:
This is not necessary because ATTDEF is always replaces by an ATTRIB entity attached to the INSERT entity.

Reply:
BTW: The attached ATTRIBs are located in the same coordinate system as the block reference, to examine the attached ATTRIBs just examine `Insert.attribs`